cloud/gcp/modules/compute/variables.tf

```diff
@@ -0,0 +1,29 @@
+variable "region" {
+  type        = string
+  description = "Default region for the db"
```

Comment:
question (blocking): this file is in the compute module, is this description accurate?

Reply:
done